sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/Monitor.java

```diff
@@ -39,20 +39,15 @@
   private class MonitorDoFn extends DoFn<T, T> {
     final Counter elementCounter = Metrics.counter(name, prefix + ".elements");
     final Counter bytesCounter = Metrics.counter(name, prefix + ".bytes");
-    final Distribution startTime = Metrics.distribution(name, prefix + ".startTime");
-    final Distribution endTime = Metrics.distribution(name, prefix + ".endTime");
-    final Distribution startTimestamp = Metrics.distribution(name, prefix + ".startTimestamp");
-    final Distribution endTimestamp = Metrics.distribution(name, prefix + ".endTimestamp");
+    final Distribution systemTimestamp = Metrics.distribution(name, prefix + ".systemTimestamp");
```

Comment:
it is not a timeStamp, it is in processing time (the wall clock) as opposed to a timestamp which is a component of an element put by the source. Please rename to "processingTime"

Reply:
please rename also the metric name